File: Euler-Java/src/Problem_05.java

public class Problem_05 {
	public static void main(String args[]){
		Problem_05 prob_05Obj = new Problem_05();
		int n = 2520;
		int Switch = 1;
		while (Switch == 1){
			Integer smallest = prob_05Obj.evenlyDivisible(n);
			if(smallest != 1){
				System.out.println(smallest);
				Switch = 0;
			}else{
				n += 2520;
			}
		}
	}
	
	public Integer evenlyDivisible(int n)
	{
		int counter = 0;
		for (int i = 1; i < 21; i++)
		{
			if(n % i == 0)
			{
				counter = counter + 1;
				if (counter == 20){
					return n;
				}
				else{
					continue;
				}
			}
			else{
				return 1;
			}
		}
		return 1;
	}
}

File: Javascript/problem_05.js
var n = 2520;
var on_off = 1;
while(on_off == 1){
	var smallest = evenlyDivisible(n);
	if(smallest != 1){
		document.write(smallest);
		on_off = 0;
	}
	else{
		n += 2520;
	}
}

function evenlyDivisible(n){
	// Checks if number (n) is evenly divisible by the numbers 1-20.
	var counter = 0;
	for(var i = 1; i <= 20; i++){
		if(n % i == 0){
			counter += 1;
			if(counter == 20){
				return n;
			}
		}
		else{
			return 1;
			}
	}
}